build/arm-tests/3d-0m-3c-ext-kp-location.json

```diff
@@ -4,7 +4,7 @@
 	"why" : "",
 	"location" : "westeurope",
 	"parameters" : {
-		"loadBalancerType":{"value":"external"},
+		"loadBalancerType":{"value":"External"},
```

Comment:
I sort of agree with change but it invalidates all existing arm automation for little gains IMO, i rather we reverse this back to `internal` and `external`